### DISCUSSION TITLE
No limits or Increase limits on shortcuts

### DISCUSSION THREAD

**Discussion by busyxiang** [dailydotdev/daily#538] [Feature Requests]
It would be nice if we are able to have more shortcuts, currently its not enough

**Comment by idoshamun** [comment]
Hi @busyxiang,

Thank you for the suggestion.
At this point, we have no plans in changing the shortcuts. 😅

**Reply by thisisujjawalbhatia** [reply]
> Hi @busyxiang,
> 
> Thank you for the suggestion. At this point, we have no plans in changing the shortcuts. 😅

hey hi, do you consider increasing the number of shortcut tabs now, I believe that if every individual has the choice to have the number of shortcuts they want to have in their dashboard, it can significantly reduce the number of tabs they need to open, it will not only increase user experience but also reduce stress and pressure you feel while having a huge number of tabs opened. It can even help in increasing overall productivity of people.

**Reply by idoshamun** [reply]
Hi @thisisujjawalbhatia,

Unfortunately not yet
